### DISCUSSION TITLE
Any advice on building Jazzer for Android OS (ART) x86 or ARM?

### DISCUSSION THREAD

**Discussion by MHA3** [CodeIntelligenceTesting/jazzer#412] [Q&A]
Hi, what build files, source code etc. would/may need to be updated to be able to build Jazzer for Android?

**Comment by ghost** [comment]
Hi @MHA3 ! I know you brought this up nearly 2-years ago...but maybe this discussion helps: https://github.com/CodeIntelligenceTesting/jazzer/discussions/413

I'd be happy to go through your question for building Jazzer for Android. We can talk here or "offline" at david.merian[at]code-intelligence[dot]com

Cheers,
Dave
